tests/parsers/fixtures/orca/unsuccessfull/aiida.out

```diff
@@ -0,0 +1,2244 @@
+Thermal Enthalpy correction       ...      0.00094421 Eh       0.59 kcal/mol
+-----------------------------------------------------------------------
+Total Enthalpy                    ...    -39.91509734 Eh
+
+
+Note: Rotational entropy computed according to Herzberg 
+Infrared and Raman Spectra, Chapter V,1, Van Nostrand Reinhold, 1945 
+Point Group:  Td, Symmetry Number:  12  
+Rotational constants in cm-1:     5.167258     5.167218     5.167200 
+
+Vibrational entropy computed according to the QRRHO of S. Grimme
+Chem.Eur.J. 2012 18 9955
+
+
+-------
+ENTROPY
+-------
+
+The entropy contributions are T*S = T*(S(el)+S(vib)+S(rot)+S(trans))
+     S(el)   - electronic entropy
+     S(vib)  - vibrational entropy
+     S(rot)  - rotational entropy
+     S(trans)- translational entropy
+The entropies will be listed as multiplied by the temperature to get
+units of energy
+
+Electronic entropy                ...      0.00000000 Eh      0.00 kcal/mol
+Vibrational entropy               ...      0.00002150 Eh      0.01 kcal/mol
+Rotational entropy                ...      0.00483862 Eh      3.04 kcal/mol
+Translational entropy             ...      0.01627961 Eh     10.22 kcal/mol
+-----------------------------------------------------------------------
+Final entropy term                ...      0.02113973 Eh     13.27 kcal/mol
+
+In case the symmetry of your molecule has not been determined correctly
+or in case you have a reason to use a different symmetry number we print 
+out the resulting rotational entropy values for sn=1,12 :
+ --------------------------------------------------------
+|  sn= 1 | S(rot)=       0.00718482 Eh      4.51 kcal/mol|
+|  sn= 2 | S(rot)=       0.00653036 Eh      4.10 kcal/mol|
+|  sn= 3 | S(rot)=       0.00614753 Eh      3.86 kcal/mol|
+|  sn= 4 | S(rot)=       0.00587590 Eh      3.69 kcal/mol|
+|  sn= 5 | S(rot)=       0.00566522 Eh      3.55 kcal/mol|
+|  sn= 6 | S(rot)=       0.00549307 Eh      3.45 kcal/mol|
+|  sn= 7 | S(rot)=       0.00534753 Eh      3.36 kcal/mol|
+|  sn= 8 | S(rot)=       0.00522145 Eh      3.28 kcal/mol|
+|  sn= 9 | S(rot)=       0.00511024 Eh      3.21 kcal/mol|
+|  sn=10 | S(rot)=       0.00501076 Eh      3.14 kcal/mol|
+|  sn=11 | S(rot)=       0.00492077 Eh      3.09 kcal/mol|
+|  sn=12 | S(rot)=       0.00483862 Eh      3.04 kcal/mol|
+ --------------------------------------------------------
+
+
+-------------------
+GIBBS FREE ENERGY
+-------------------
+
+The Gibbs free energy is G = H - T*S
+
+Total enthalpy                    ...    -39.91509734 Eh 
+Total entropy correction          ...     -0.02113973 Eh    -13.27 kcal/mol
+-----------------------------------------------------------------------
+Final Gibbs free energy         ...    -39.93623707 Eh
+
+For completeness - the Gibbs free energy minus the electronic energy
+G-E(el)                           ...      0.03173838 Eh     19.92 kcal/mol
+
+NOTE(danielhollas): The end of this file was purposfully deleted,
```

Comment:
If the point of this test is to just check that you get `ERROR_CALCULATION_UNSUCCESSFULL` why do you need all this output? Unless you explicitly want to test that you still parse some explicit partial output, you may want to cut a lot more output. This is from the perspective of not bloating the repo too much. As you start adding more tests, if each tests adds thousands of lines, this may start to grow the repo unnecessarily.

Reply:
Yes, the point of this was to indeed verify that the output was still parsed in the same way as in the default test.
Agreed about the repo bloat. I'd probably keep this one anyway if @pzarabadip agrees.